buildkite/culprit_finder.py

```diff
@@ -174,10 +183,25 @@ def main(argv=None):
     if args.subparsers_name == "culprit_finder":
         try:
             project_name = os.environ["PROJECT_NAME"]
+
             # For old config file, we can still set PLATFORM_NAME as task name.
-            task_name = os.environ.get("PLATFORM_NAME") or os.environ["TASK_NAME"]
-            good_bazel_commit = os.environ["GOOD_BAZEL_COMMIT"]
-            bad_bazel_commit = os.environ["BAD_BAZEL_COMMIT"]
+            if "PLATFORM_NAME" in os.environ or "TASK_NAME" in os.environ:
+                tasks = [os.environ.get("PLATFORM_NAME") or os.environ["TASK_NAME"]]
+            else:
+                tasks = get_tasks(project_name)
+
+            if "GOOD_BAZEL_COMMIT" in os.environ:
```

Comment:
Nit: I'd prefer

good_bazel_commit = os.environ.get("GOOD_BAZEL_COMMIT")
if not good_bazel_commit:
  # If GOOD....

Reply:
Done.